docs/components.md

```diff
@@ -86,6 +86,64 @@ Options:
         pentagon add vpc -D vpc_name="pentagon-vpc" -D vpc_cidr_base="172.20" -D aws_availability_zones="ap-northeast-1a, ap-northeast-1c" -D aws_availability_zone_count = "2" -D aws_region = "ap-northeast-1"
         ```
         
+### kops.cluster
+
+- add: 
+    - Creates yml files in  `./<cluster_name>/` compiled from the data passed in.
+    - `bash ./<cluster_name>/kops.sh` will create the cluster as configured.
+    - Argument/ ConfigFile keys:
+      - `additional_policies`: Additional IAM policies to inflict upon the cluster
```

Comment:
Please include an example of this in the config file as well as it is likely complex enough to get right that it will likely be wrong for the first tries by a user.

Reply:
Example looks nice!